llvm/test/tools/yaml2obj/XCOFF/offset-check.yaml

```diff
@@ -0,0 +1,156 @@
+## Case 1: an error is reported when FileOffsetToData is too small
+## RUN: not yaml2obj --docnum=1 %s -o %t 2>&1 | \
+## RUN:   FileCheck %s --check-prefix=ERROR1
+#
+# ERROR1: error: current file offset (60) is bigger than the specified FileOffsetToData for the .text section (48)
```

Comment:
Nit: Include a blank line between your check and the YAML for improved readability. Applies throughout.